arcgis-ios-sdk-samples/Features/Toggle between feature request modes/README.metadata.json

```diff
@@ -17,7 +17,11 @@
         "AGSServiceFeatureTable.featureRequestMode",
         "AGSServiceFeatureTable.populateFromService(with:clearCache:outFields:completion:)"
     ],
-    "redirect_from": [],
+    "redirect_from": [
```

Comment:
```suggestion
    "redirectFrom": [
```

Reply:
in the migration to Gatsby we changed this to `redirectFrom` see https://github.com/ArcGIS/afd-core/pull/1157 - - but we need to merge this suggestion and build a -preview and test it first @ThadT

Reply:
Just spoke with the dev team on this my suggestion won't change anything. Creating a separate issue to address this in https://github.com/ArcGIS/afd-native-apis-doc/issues

Reply:
https://github.com/ArcGIS/afd-native-apis-doc/issues/1890

Reply:
Thanks, @jf990

Reply:
With the recent merge of https://github.com/ArcGIS/afd-native-apis-doc/pull/1898 , is it OK to proceed with this PR?

Reply:
i think we should move forward with it